cmd/fence-provision-manager/cmd/fence.go

```diff
@@ -50,14 +68,14 @@ func fence(cmd *cobra.Command, args []string) (err error) {
 	if err != nil {
```

Comment:
Make sure we document how to pass in username/password
Side-effects:
1. all usernames & passwords must now be part of a secret
2. only usernames & passwords can be considered sensitive

Not a huge fan of 1) but we need to make sure we document it well.
Historically some customers would see 2) as problematic.

Reply:
In general, I can change this to `Path to the secret that contains fencing agent sensitive information`, and I assume that it will have username and password fields, but we assumed the same under `secret-handler.sh` file, so I do not see a problem here.

Reply:
So for every field, check to see if its available from the secret path and if not look in the parameter list?  That would work.

To chose a bad example, an admin might decide that the IPMI server's address is top secret.